Library/Homebrew/cmd/vendor-install.sh

```diff
@@ -174,6 +174,18 @@ install() {
   tar "$tar_args" "$CACHED_LOCATION"
   safe_cd "$VENDOR_DIR/portable-$VENDOR_NAME"
 
+  if [[ -n $HOMEBREW_LINUX ]]; then
+    if [[ -x "$HOMEBREW_PREFIX"/opt/patchelf/bin/patchelf ]]; then
```

Comment:
Should be an error if this doesn't exist, right? How is this going to handle when we're using `patchelf.rb` instead?